Makefile

```diff
@@ -65,10 +65,8 @@ rpm: tarball ver2spec  ## Build RPMs
 srpm: tarball ver2spec  ## Build SRPM
 	rpmbuild --define '_topdir $(TMP)' -bs tmt.spec
 
-_deps:  # Minimal dependencies (common for 'deps' and 'develop' targets)
```

Comment:
This target should be running the newly added script, and both targets bellow would still depend on this target to do the job rather than running it on their own. If there is a "install minimal dependencies" action, it should be a dependable target.